### ISSUE TITLE
create Chip-Group components

### ISSUE THREAD

**Issue by b1aserlu** [swisspost/design-system#2199]
Blocked by #2198 

Create a Wrapper for the new Chip component #2198 according to the specifiations in figma and in this description which you can also find in the orignal issue #1864. There should be a HTML/CSS version as well as a web-component just like the chip component. #2198 

The Chips have a radio button and a checkbox button version.

## General

```[tasklist]
- [ ] radio group (aka "Filter")
- [ ] checkbox group
```
## Filter type styles to change
```[tasklist]
### General
- [ ] Gap between items should be `size-mini` (8px)
- [ ] Check if line-height from design is appropriate 150%. Or use whatever is best for the dev.
```

**Comment by gfellerph** [comment]
- [ ] Update description to include info about creating HTML/CSS version and web component version

**Comment by gfellerph** [comment]
Closing as already done with utility classes
